spec/features/admin/order_cycles_spec.rb

```diff
@@ -147,9 +147,8 @@
     page.should have_content 'Your order cycle has been created.'
 
     page.should have_selector 'a', text: 'Plums & Avos'
-
-    page.should have_selector "input[value='2040-11-06 06:00:00 +1100']"
-    page.should have_selector "input[value='2040-11-13 17:00:00 +1100']"
+    page.should have_selector "input[value='#{Time.zone.local(2040, 11, 06, 06, 00, 00)}']"
```

Comment:
Cool. Since these are repeating a few times, it would be even better to define them in the beginning:
```rb
let(:meaningful_time) { Time.zone.local(2040, 11, 06, 06, 00, 00) }
```

What do you think?